src/types/formatted/healthCareLocalFacility.ts

```diff
@@ -43,5 +44,101 @@ export type HealthCareLocalFacility = PublishedEntity & {
   socialLinks: FacilitySocialLinksProps
   lovellVariant?: LovellChildVariant
   lovellSwitchPath?: string
-  healthServices: NodeHealthCareLocalHealthService[] // TODO: Format these
+  healthServices: FormattedVAMCFacilityHealthServices[]
+}
+
+/** Represents the "single" object containing service-related fields. */
+interface VamcFacilityServiceLocation {
```

Comment:
Inverse of the above: We may want to rename this to be more generic if we discover it's useful for VHA and VBA facilities.